openmmtools/multistate/multistatesampler.py

```diff
@@ -49,6 +49,12 @@
 
 logger = logging.getLogger(__name__)
 
+# ==============================================================================
+# Warn that API is experimental
+# ==============================================================================
+
+import warnings
+warnings.warn('Warning: openmmtools.multistate API is experimental')
```

Comment:
We can probably remove this if we have it in `multistate/__init__.py`

Reply:
Whoops! I had meant to do that!